Automattic/jetpack
# Module-active checks should always return modules as active on WordPress.com simple

Fixes #34231, closes https://github.com/Automattic/greenhouse/issues/1959

## Proposed changes:

* As outlined in #34231 we should always return true on module active checks when running Jetpack on WordPress.com

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Note: Original context for this was #34229. There might be other tests that need to be performed in other areas of the plugin.
* Apply PR as a diff on wpcom and sandbox a simple site with a plan that includes VideoPress
* Create a subscription plan. For more info check https://github.com/Automattic/jetpack/pull/32154
* Create a post, add a subscriber-only block and a videopress private video block.
* View as a subscriber. It should work.
